DUI3/Speckle.Connectors.DUI/Url.cs

```diff
@@ -12,7 +12,8 @@
 /// </remarks>
 public static class Url
 {
-  public static readonly Uri Netlify = new("https://boisterous-douhua-e3cefb.netlify.app/");
+  //old beta
+  public static readonly Uri Netlify = new("https://6671ddd20235b90008dbc453--boisterous-douhua-e3cefb.netlify.app/");
```

Comment:
Is this no longer in he XAML at all? We will need to be careful not to accidentally take this forever. Does @oguzhankoral have  aticket now for fixing the CEF 60 something support? If so can we have a subtask and drop it here to update the URL back to the right one plz?